docs/pages/API/View/index.js

```diff
@@ -0,0 +1,387 @@
+import React, { PureComponent } from 'react'
+import { Prism as SyntaxHighlighter } from 'react-syntax-highlighter'
+import { duotoneEarth } from 'react-syntax-highlighter/dist/styles/prism'
+import { Text } from '../../../../src'
+import ViewBasicSpecimen from '../../../specimens/ViewBasic'
+import style from '../style.less'
+
```

Comment:
Would be helpful to know which props are (not) responsive

Reply:
Updated